build/build.ts

```diff
@@ -58,7 +58,6 @@ const addExtension = (extension: string = '.js', fileExtension: string = '.ts'):
 })
 
 const commonOptions: BuildOptions = {
-  watch: isWatch,
```

Comment:
I added this because this line causes error `Invalid option in build() call: "watch"`.

I couldn't find that Bun docs referring to the `watch` option, and [its only suggestion](https://bun.sh/docs/bundler#watch-mode) is to use `--watch` option in command.

Reply:
@fel1x-developer 

I can't get the error. How can I reproduce it? The `watch` is for `esbuild`, unrelated to the Bun.